python/ray/train/tensorflow/tensorflow_trainer.py

```diff
@@ -85,7 +85,7 @@ def train_loop_per_worker():
 
     Example:
 
-    .. code-block:: python
+    .. testcode::
```

Comment:
We need at least one GPU test, or else Bazel fails and complains that no tests were found.